src/ui/blueprints/BlueprintEditor.tsx

```diff
@@ -40,10 +46,12 @@ export function PostEditor( props: Props ) {
 			if ( ! fieldWaitingForSelection || ! lastClickedElement ) {
 				return;
 			}
+			const selector = 'TODO';
```

Comment:
You could use the code from the tours plugin here: https://github.com/Automattic/tours/blob/trunk/assets/js/tour-step-editor.js#L161-L196